.github/actions/phoenix-build/action.yml

```diff
@@ -35,19 +43,26 @@ inputs:
     description: 'Parameters to the main build script - 5th param'
     default: ''
     required: false
+  param6:
+    description: 'Parameters to the main build script - 6th param'
+    default: ''
+    required: false
 
 # action runner
 runs:
   using: 'docker'
   image: 'phoenixrtos/build'
   env:
     TARGET: ${{ inputs.target }}
+    CONSOLE: ${{ inputs.console }}
```

Comment:
this is not set anywhere, but I assume it might be in the future

Reply:
You are right - it is not necessary for now. I removed it